projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -483,7 +489,7 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      * ```
      */
     @Output()
-    public onSearchInput = new EventEmitter();
+    public onSearchInput = new EventEmitter<IComboSearchInputEventArgs>();
```

Comment:
Let's extend also the IBaseEventArgs (like the `onAddition` event). Also the `searchTerm` should be renamed to 'value'